### PR TITLE
Update clients.md

### PR DESCRIPTION
**Pull request by yosefd**
Fix instructions for using SSMS

### CODE REVIEW

**Comment by PRMerger8** [comment]
@yosefd : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: data-explorer/kusto/api/tds/clients.md [hunk 1]
```diff
@@ -213,8 +213,8 @@ Create a connection to the Azure Data Explorer database.
 ## Microsoft SQL Server Management Studio (v18.x)
 
 1. Select **Connect**, and then **Database Engine** under **Object Explorer**.
-1. Specify the name of Azure Data Explorer cluster as a server name. For example, *mykusto.kusto.windows.net*.
-1. Set **Active Directory - Integrated** for authentication.
+1. Specify the name of Azure Data Explorer cluster as a server name. For example, *mykusto.region.kusto.windows.net*.
+1. Set **Azure Active Directory - Universal with MFA** for authentication and specify user name.
```
Comment (shsagir) [inline data-explorer/kusto/api/tds/clients.md:217]:
```suggestion
1. Set **Azure Active Directory - Universal with MFA** for authentication and specify the username.
```

**Comment by PRMerger16** [comment]
@shsagir : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by shsagir — APPROVED** [review]

**Comment by yosefd** [comment]
#signoff

**Comment by yosefd** [comment]
#sign-off

**Comment by shsagir** [comment]
#sign-off

**Pull request merged**